deploy.yaml

```diff
@@ -32,11 +32,11 @@ spec:
           procMount: Unmasked
         resources:
           limits:
-            cpu: 50m
-            memory: 50Mi
+            cpu: 200m
+            memory: 200Mi
           requests:
-            cpu: 50m
-            memory: 50Mi
+            cpu: 200m
```

Comment:
we should keep request small, as this may run everywhere.

Reply:
The apt-get install for ftrace-cmd was getting OOMd due to small memory request, so that's why I bumped this. Perhaps keep request small but increase limit? WDYT?

Reply:
We could just leave limit off (let the system deal w/ overflow)